jbpm-console-ng-process-runtime-forms/jbpm-console-ng-process-runtime-forms-client/src/main/java/org/jbpm/console/ng/pr/forms/client/editors/quicknewinstance/QuickNewProcessInstancePopup.java

```diff
@@ -178,11 +180,11 @@ protected void loadFormValues() {
         final Map<String, List<String>> dropDowns = new HashMap<String, List<String>>();
         processDefinitionsListBox.clear();
         processDeploymentIdListBox.clear();
-        currentFilter = new PortableQueryFilter( 0,
-                                                 10,
-                                                 false, "",
-                                                 "",
-                                                 true );
+        currentFilter = new PortableQueryFilter(0,
+                                                PAGE_SIZE_UNLIMITED,
+                                                false, "",
+                                                FIELD_ID_PROCESSNAME,
```

Comment:
I think this sort is not in place on master, would be good to send this PR in there as well.